### ISSUE TITLE
Question about the Loss functions (L2 norm?)

### ISSUE THREAD

**Issue by htakatsuka** [manigalati/usad#9]
Hello. May I ask about the loss functions inside the training_step?
The paper states that ||x||2 denotes L2 Norm. As I understand, L2 norm is the "square root of sum of squares of elements inside x".
However, the loss of the training step seems to be using MSE (mean((batch-w3)**2) etc).
Was this a change from the paper or did I not understand L2 norm correctly?
Thanks in advance!

**Comment by htakatsuka** [comment]
This issue was only a question. No one has answered me until now so I will close this. Thank you.
